docs/terms/evaluation_parameter.md

```diff
@@ -1,15 +1,132 @@
 ---
 id: evaluation_parameter
 title: Evaluation Parameter
-hoverText: A parameter used during Validation of an Expectation to reference simple expressions or previousl generated metrics.
+hoverText: A connector to store and retrieve information about parameters used during Validation of an Expectation which reference simple expressions or previously generated metrics.
```

Comment:
I'm wary of using the word "connector" given it's association with DataConnectors.

Is there a more suitable word we can use here?

Reply:
I tend to agree with @cdkini -- what about something along the lines of: `A dynamic value used during Validation of an Expectation which is populated by evaluating simple expressions or by referencing previously generated metrics.`

Reply:
I'll update this with "A dynamic value used during Validation of an Expectation which is populated by evaluating simple expressions or by referencing previously generated metrics." as suggested.